src/integration.ts

```diff
@@ -109,16 +124,17 @@ export class ProfilingIntegration implements Integration {
 
         const profilesToAddToEnvelope: Profile[] = [];
 
-        for (let i = 0; i < profiledTransactionEvents.length; i++) {
-          const profiledTransaction = profiledTransactionEvents[i];
-          const profile_id = profiledTransaction?.contexts?.['profile']?.['profile_id'];
+        for (const profiledTransaction of profiledTransactionEvents) {
+          const profileContext = profiledTransaction.contexts && profiledTransaction.contexts['profile']
```

Comment:
I think we should revert these changes. `node-profiling` supports Node.js 14 and up, which supports optional chaining: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
This is actually a rule in the tsconfig and afaik we do this because of bundle size constraints (optional chaining doesn't minify well)

Reply:
We disable these lints in `@sentry/node` though because bundle size is less of a concern there:
https://github.com/getsentry/sentry-javascript/blob/develop/packages/node/.eslintrc.js

Reply:
Aghh, nice find @timfish, I missed that! Will revert these changes